### ISSUE TITLE
AttributeError: 'Generalized_RCNN' object has no attribute 'detectron_weight_mapping'

### ISSUE THREAD

**Issue by zhenwenYANG** [ppengtang/pcl.pytorch#84]
This problem occurs when I reappear, how to solve it?

**Comment by Galaxy2233** [comment]
Have you solve this problem? I find it occurs too

**Comment by zhenwenYANG** [comment]
No, I added the attention module before and then this error occurred. Later, after deleting the added part, it returned to normal



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ppengtang/pcl.pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月20日(星期四) 晚上9:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ppengtang/pcl.pytorch] AttributeError: 'Generalized_RCNN' object has no attribute 'detectron_weight_mapping' (Issue #84)





 
Have you solve this problem? I find it occurs too

**Comment by Galaxy2233** [comment]
真是一个悲伤的故事

**Comment by bityangke** [comment]
you need to add weight mapping for newly added modules
